totalspineseg/inference.py

```diff
@@ -698,5 +704,60 @@ def main():
             },
         )
 
+    if not iso:
+        if not quiet: print('\n' 'Resampling step1_output to the input images space:')
+        transform_seg2image_mp(
+            input_path,
+            output_path / 'step1_output',
+            output_path / 'step1_output',
+            image_suffix = '',
+            overwrite=True,
+            max_workers=max_workers,
+            quiet=quiet,
+        )
+        if not quiet: print('\n' 'Resampling step1_cord to the input images space:')
+        transform_seg2image_mp(
+            input_path,
+            output_path / 'step1_cord',
+            output_path / 'step1_cord',
+            image_suffix = '',
+            interpolation = 'linear',
```

Comment:
Shouldn't we use nearest neighbour ? To keep integer values ?

Reply:
No, because this is soft segmentation